composer.json

```
@@ -17,9 +17,9 @@
     ],
     "require": {
         "php": "^7.4|^8.0",
```

We can safely declare that we are only compatible with PHP 8 or above

```suggestion
        "php": "^8.0",
```